lib/tasks/temp.rake

```diff
@@ -16,6 +16,44 @@ namespace :temp do
     end
   end
 
+  def backload_overdue(event_type, verbose)
```

Comment:
Reading through this in more detail, aside from the ability to write some output per event, is there any functional difference between what this is doing and just calling `InfoRequest.log_overdue_events` or `InfoRequest.log_very_overdue_events`?